samples/tokenCredentialSamples/ClientCredentialFlow/typescript/secrets.sample.ts

```diff
@@ -0,0 +1,4 @@
+export const tenantId = "<TENANT_ID>";
+export const clientId = "<CLIENT_ID>";
+export const clientSecret = "<CLIENT_SECRET>";
+export const scopes = "<SCOPE>";
```

Comment:
What happens when one needs more than one scope? What type of delimiter can they use?

Reply:
From other samples, it looks like this needs to be an array. Sample data implies that it is a string for a single scope.

Reply:
scopes could be an array or a URL that is a string. For example - scope while using the sample would be - "https://graph.microsoft.com/.default".  The secrets.sample files should be renamed and the placeholders should be replaced with the values when someone wants to run the sample.